build/buildPackage.sh

```diff
@@ -127,6 +126,15 @@ manifest="
 
 "
 
+if [ "$#" -lt "1" ]; then
```

Comment:
Is this definitely right? My bash isn't all that strong, but it looks like we're only checking `$1` if the number of arguments is 0?

Since the primary purpose of this repo is to package _all_ the dependencies for Gaffer, I'd also like to keep the default behaviour for the flagless invocation as it is, so the default builds the same package it always has...